.npmignore

```diff
@@ -0,0 +1 @@
+src/
```

Comment:
I think having .gitignore in .npmignore is a good idea, for the reasons you mentioned.  It's too bad we don't have a system of #include or something inside these ignore files, or could say "use .gitignore, but also ignore these files" in package.json.

I suspect that we'll end up having to have a node module that has a helper script to set up modules correctly that's run as an npm script instead of just being able to automatically integrate these things.